components/brave_wallet_ui/common/async/__mocks__/bridge.ts

```diff
@@ -465,11 +460,6 @@ export class MockedWalletApiProxy {
         ? { mnemonic: mockedMnemonic }
         : { mnemonic: '' }
     },
-    getChecksumEthAddress: async (address) => {
-      return {
-        checksumAddress: address.toLocaleLowerCase()
-      }
-    },
```

Comment:
used by mocked version of `getEthAddressChecksum` in `common/slices/endpoints/address.endpoints.ts`